README.textile

```diff
@@ -114,6 +115,66 @@ installation and configuration. Information about reporting problems and
 getting help for ImageMagick is available at the "ImageMagick web site":http://www.imagemagick.org
 or the "ImageMagick Forum":http://www.imagemagick.org/discourse-server.
 
+h2(#dev-setup). Development Setup
+
+In order to minimize issues on your local machine, we recommend that you make
+use of a "Vagrant installation":https://github.com/tjschuck/rake-compiler-dev-box.
+
+Steps to get up and running with a passing build are as follows:
+
+h4. 1) set up the Vagrant environment
+
+<pre>
+git clone https://github.com/tjschuck/rake-compiler-dev-box.git
+cd rake-compiler-dev-box
+vagrant up
+</pre>
+
+This last part will probably take a while as it has to download an Ubuntu image
+and configure it. If there is an error during this process, you may need to
+reboot your computer and enable virtualization in your BIOS settings. You may
+also want to add the following line to the Vagrantfile for convenience:
+
+<pre>
+config.ssh.forward_agent = true
+</pre>
+
+You'll need to run `vagrant reload` for the settings to take effect.
+
+h4. 2) log in to the vagrant box and clone RMagick
+
+<pre>
+vagrant ssh
+git clone https://github.com/gemhome/rmagick.git # or your fork
```

Comment:
I prefer to clone it, edit it, and commit/push changes outside of the ssh session.  I only use the ssh session to run tests (and install other rubies, packages etc).  If you commit within the session, I don't think it forwards your git config.  Try `git config --global --list | grep user`